py/configu/commands/eval_command.py

```diff
@@ -135,16 +135,13 @@ def _eval_from_store_set(
                 value["result"]["value"] = store_config.value
         return result
 
-    def _eval_from_schema(
-        self, result: EvalCommandReturn
-    ) -> EvalCommandReturn:
+    @staticmethod
```

Comment:
why is it marked as staticmethod?

Reply:
semantics. it doesn't use `self` (JS:`this`). it's also marked "private" so even though it's static it's not for outside use.
if it bothers you we can lose it. it's just semantics.